### ISSUE TITLE
Extra thumbnail in playback

### ISSUE THREAD

**Issue by mojtaba82** [bigbluebutton/bigbluebutton#14640]

**Comment by pedrobmarin** [comment]
The playback will show the last slide thumbnail because it does not test if the slide has a valid timestamp over the video duration.
![Screenshot 2022-03-21 at 12-54-47 Playback](https://user-images.githubusercontent.com/1778398/159300083-6d1fc64d-e866-437c-8dcb-d31f48bd0bc9.png)

**Comment by danielpetri1** [comment]
Hi, I believe this is a potential duplicate of #11310.
This should be fixed once PR #13836 is merged.
